JPA + JSF tests enabled by this pr seem to pass for me locally.  

The following test fails:
```
mvn install -Dts.ee9 -Dtest=org.wildfly.test.integration.web.annotationsmodule.EarModuleDeploymentTestCase

Failures: 
[ERROR]   EarModuleDeploymentTestCase.testEjbScan:155 expected:<200> but was:<500>
[ERROR]   EarModuleDeploymentTestCase.testEjbScanInSubdeployment:191 expected:<200> but was:<500>
[ERROR]   EarModuleDeploymentTestCase.testServlet:119 expected:<200> but was:<404>
[ERROR]   EarModuleDeploymentTestCase.testServletInAdditinalJar:137 expected:<200> but was:<404>
[ERROR]   EarModuleDeploymentTestCase.testServletInDeployment:173 expected:<200> but was:<404>
[INFO] 
[ERROR] Tests run: 5, Failures: 5, Errors: 0, Skipped: 0
```

--- a/testsuite/integration/basic/pom.xml
+++ b/testsuite/integration/basic/pom.xml
@@ -3465,9 +3465,6 @@
                                         <!-- Requires the Arquillian Servlet 3.0 protocol to be overridden to Servlet 5.0 -->
                                         <exclude>org/jboss/as/test/integration/logging/config/LoggingProfileSharedTestCase.java</exclude>
 
-                                        <!-- Excluded because Hibernate doesn't conform to CDI 4 deprecated method removals -->
-                                        <exclude>org/jboss/as/test/integration/ee/injection/support/jpa/EntityListenerInjectionSupportTestCase.java</exclude>
-
                                         <!-- With EE9, we use Hibernate Search 6 instead of 5,
                                              and Hibernate Search 6 exposes a different (backwards-incompatible) API,
                                              so Hibernate Search 5 tests cannot work.
@@ -3550,12 +3547,6 @@
                                         <exclude>org/jboss/as/test/integration/management/deploy/runtime/JaxrsRuntimeNameTestCase.java</exclude><!-- test expects javax strings in mgmt api responses -->
                                         <exclude>org/jboss/as/test/integration/weld/ejb/SessionObjectReferenceTestCase.java</exclude>
                                         <exclude>org/wildfly/test/integration/web/annotationsmodule/EarModuleDeploymentTestCase.java</exclude>
-
-                                        <!-- Excluded because Hibernate doesn't conform to CDI 4 deprecated method removals -->
-                                        <exclude>org/jboss/as/test/integration/ee/injection/support/jpa/EntityListenerInjectionSupportTestCase.java</exclude>
-                                        <exclude>org/jboss/as/test/integration/jpa/initializeinorder/InitializeInOrderTestCase.java</exclude>
-                                        <exclude>org/jboss/as/test/integration/jpa/entitylistener/EntityListenersTestCase.java</exclude>
-                                        <exclude>org/jboss/as/test/integration/ee/injection/support/jpa/beanmanager/EntityListenerBeanManagerInjectionTestCase.java</exclude>
 
                                         <!-- With EE9, we use Hibernate Search 6 instead of 5,
                                              and Hibernate Search 6 exposes a different (backwards-incompatible) API,

